enterprise/server/remote_execution/snaploader/snaploader.go

```diff
@@ -22,6 +21,22 @@ const (
 	ManifestFileName = "manifest.json"
 )
 
+// NewKey returns the cache key that can be used to look up the snapshot
+// manifest.
+// TODO(bduffany): once runners are capable of sharing snapshots, remove
+// runnerID from the digest hash.
+// TODO(bduffany): use ResourceName instead of digest and incorporate
+// remote_instance_name. The instance name is currently implicit in the runnerID
+// since we only match tasks to runners if their instance name matches the
+// runner's instance name. But once we have snapshot sharing, runnerID will no
+// longer be part of the snapshot key.
+func NewKey(configurationHash, runnerID string) *repb.Digest {
```

Comment:
FYI, I am calling this "Key" instead of "Digest", because in a later PR I'm planning to add a `Key` struct which contains `ResourceName` and `RunnerID` (`RunnerID` will be used to prevent concurrent access by multiple runners, and is only relevant to the local filecache, not the remote cache).